Desktop/data/datasetpackage.cpp

```diff
@@ -1532,29 +1531,16 @@ void DataSetPackage::initializeComputedColumns()
 }
 
 
-stringvec DataSetPackage::getColumnNames()
+ColumnEncoder::colTypeMap DataSetPackage::getColumnNamesWithTypes()
```

Comment:
So you renamed and moved the `getColumnTypesMap` around to become `getColumnNamesWithTypes`? :p